File: Lista_01_POO/Viagem/QuestaoViagem.py
class Viagem:
    def __init__(self):
        self.distancia = 0
        self.tempo = 0
    
    def velocidade(self):
        return self.distancia / self.tempo
    

v = Viagem()

v.distancia = int(input("Informe a distância: "))
v.tempo = int(input("Digite o tempo: "))
print(f"Velocidade média = {v.velocidade():.2f}")

File: POOLists/Lista02/Viagem/questaoViagem.java
import java.util.Scanner;

public class questaoViagem {
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);
        Viagem v = new Viagem();

        System.out.println("Digite a distancia percorrida: ");
        v.setDistancia(input.nextDouble());
        System.out.println("Digite o tempo: (ex: 1,5)");
        v.setTempo(input.nextDouble());

        System.out.println(String.format("Velocidade media = %.2f", v.velocidadeMedia()));
    
        input.close();
    }
    
}
